riscv/triggers.cc

```diff
@@ -151,6 +151,8 @@ void mcontrol_t::tdata1_write(processor_t * const proc, const reg_t val, const b
       match = MATCH_EQUAL;
       break;
   }
+  vs = 0;
+  vu = 0;
```

Comment:
No need for these two lines; they are both initialized by the constructor and will never change.